ocs_ci/utility/performance_dashboard.py

```diff
@@ -39,21 +38,21 @@ def initialize_data():
     Returns:
         dict: A dictionary contains the data to push to the dashboard
     """
-    csv = ocp.OCP(kind='csv', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
     worker_type = get_typed_nodes(num_of_nodes=1)[0].data['metadata'][
         'labels'
     ]['beta.kubernetes.io/instance-type']
 
-    csv_vers = csv.get()['items'][0]['spec']['version'][:-3].split("-")
-    min_version = csv_vers[0]
-    build_id = csv_vers[1]
-    ocs_ver = ".".join(min_version.split(".")[:-1])
+    (ocs_ver_info, _) = get_ocs_version()
```

Comment:
We don't need parenthesis here.

```suggestion
    ocs_ver_info, _ = get_ocs_version()
```

Reply:
@clacroix12 I am going to fix this in next version, want to unblock some testing for now.